src/css-reset.module.css

```diff
@@ -2,12 +2,12 @@
   font-family: var(--font-family);
   font-size: var(--font-size);
   color: var(--font-color);
-  box-sizing: border-box;
 }
 
+.ventura,
 .ventura *,
 .ventura *:before,
 .ventura *:after {
   box-sizing: border-box;
-  font-family: var(--font-family);
```

Comment:
Is there a reason for this? Or is this some merge conflict problem?

Reply:
I removed it because font-family is automatically inherited on all components. But while I am saying this, is this also true on the Button component?

Reply:
In the case of our Button component, the css reset class `.ventura` will be set on the button. So the user agent font-family on the Button will be overwritten by the font-family from `.ventura`